NuGet.Config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What's the resolution of this file?

Reply:
I would say leave it included. If it is ignored, you don't notice necessary changes easily. If it is included, a change draws attention automatically. Examples of inclusions:
https://github.com/xamarin/xamarin-android/pull/2903
https://github.com/davidortinau/Xappy/pull/39